.github/workflows/linting.yml

```
@@ -34,7 +34,7 @@ jobs:
         uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
 
       - name: Install Nextflow
-        uses: nf-core/setup-nextflow@v2
+        uses: nf-core/setup-nextflow@v2.0.0
```

```suggestion
        uses: nf-core/setup-nextflow@v57267e6c67300b3e82e0850adc6df50be8e16a26
```

let's see if we pin it to the merge with the hot-fix